scripts/components/PaymentForm.jsx

```diff
@@ -0,0 +1,112 @@
+var React = require('react');
+var moment = require('moment');
+var Api = require('../utils/Api.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var mui = require('material-ui');
+var DatePicker = mui.DatePicker;
+var TextField = mui.TextField;
+var RaisedButton = mui.RaisedButton;
+var Paper = mui.Paper;
+
+var PaymentForm = React.createClass({
+  getInitialState() {
+    return {
+      date: '', // user entered date
+      property: '', // user entered property address
+      amount: '', // user entered amount
+    }
+  },
+
+  propTypes: {
+        newDataAdded:   React.PropTypes.func,
+  },
+
+  // Capture the input field state after each keypress.
+  onChange(field, event) {
+    this.setState({ [field]: event.target.value });
+  },
+
+  // Handle the form submission event when the user adds new payment.
+  onSubmit(event) {
+    event.preventDefault();
+    event.stopPropagation();
+
+      // API call to add payment.
+    Api.addPayment({
+      data: {
+        date: this.refs.DatePicker.getDate(),
+        property: this.state.property,
+        amount: this.state.amount
+      },
+      callback: (err, response) => {
+        if (err) {
+          return;
+        }
+        // Clear the form
+        this.setState({
+          date: "",
+          property: "",
+          amount: ""
+        });
+
+        this.props.newDataAdded();
+      }
+    });
+
+  },
+
+  render() {
+    var { date, property, amount } = this.state;
+    var errorMessage;
+    var currentDate = new Date();
+    return (
+      <div style={style.formContainer}>
+        <h2 style={style.heading}>Make a Payment </h2>
+        <Paper zDepth={1}>
+          <form style={style.form} onSubmit={this.onSubmit}>
+            <DatePicker
+              name="Date"
+              ref="DatePicker"
+              onChange={this._handleChange}
+              floatingLabelText="Date"/>
+            <TextField
```

Comment:
It doesn't make sense for the tenant to input the property name. The tenant rent's a single property. We know which property this is so we should be doing it automatically. In addition, the tenant's account only ever displays information for a single property. From a user perspective they would go `why the hell do I have to enter the property??? they know what property I rent...`